### ISSUE TITLE
Screen sharing background completely dark after installing

### ISSUE THREAD

**Issue by mawaldne** [jszklarz-haventec/makeitdark#10]
Hey, thanks for making this script, its awesome!

I'm having an issue on Slack Native app on MacOS Mojave. 

After installing the makeitdark theme, during screen sharing the entire screen goes completely dark. You can't see anything in screen sharing.  If you click around with the mouse or Command-Tab to other windows you can see the window again. But then if someone draws on your screen it all goes dark again. 

If I uninstall makeitdark screen sharing is fine. 

I'm playing around trying to figure out if theres something in the CSS thats causing it.  If I figure it out I'll post here. If anyone knows a solution please let me know!

**Comment by mawaldne** [comment]
While I dont think this is an actual fix, its good enough for me for now. 

I ended up using the CSS file from this project: https://github.com/widget-/slack-black-theme

I just changed the JS that gets copied by the makeitdark.py script and pointed it to their CSS. Screen sharing works again. 

Note: Apparently theres a bug with their latest CSS, so I used the CSS mentioned in this issue: https://github.com/widget-/slack-black-theme/issues/62#issuecomment-410256441 eventually this will get merged into master.

Thanks again for making this script!
